enterprise/dev/ci/internal/ci/operations.go

```diff
@@ -679,7 +679,9 @@ func buildCandidateDockerImage(app, version, tag string) operations.Operation {
 			// Retag the local image for dev registry
 			bk.Cmd(fmt.Sprintf("docker tag %s %s", localImage, devImage)),
 			// Publish tagged image
-			bk.Cmd(fmt.Sprintf("docker push %s", devImage)),
+			bk.Cmd(fmt.Sprintf("docker push %s || exit 222", devImage)),
```

Comment:
nit: it would help to use an exit code different than 222 that we always use for soft failures, which is not the case here. 
```suggestion
			bk.Cmd(fmt.Sprintf("docker push %s || exit 10", devImage)),
			// Retry in case of flakes when pushing
			bk.AutomaticRetryStatus(3, 10),
```

Reply:
Good point!